### ISSUE TITLE
Missing transcriptomes

### ISSUE THREAD

**Issue by starpharm** [pachterlab/kallisto#143]
The getting started example (https://pachterlab.github.io/kallisto/10xstarting.html) says to download the combined human-mouse transcriptome but the link it gives (http://pachterlab.github.io/kallisto/transcriptomes/) does not exist.

**Comment by miguelroboso** [comment]
Yeah, I was wondering the same. I downloaded a FASTA from Ensembl, but the transcript names do not match... it would be great if there were some more documentation about this.

**Comment by mschilli87** [comment]
It seems the Download section used to contain a [link](http://bio.math.berkeley.edu/kallisto/transcriptomes/) to transcriptomes that was removed as dead earlier this year: https://github.com/pachterlab/kallisto/commit/868a6023a50bdc887909f0fb404c99458d6f9eb2.
Notably, following this link one gets referred from `bio.math.berkeley.edu` to `pachterlab.github.io` and thus the same page used in the example.

---
I also tried

```sh
git clone https://github.com/pachterlab/kallisto
cd kallisto
git checkout gh-pages
git log -- transcriptomes.md
git whatchanged | grep ':.* transcriptomes.md$'
```

which came out empty.
So it seems the site referred to in the manual never existed in this repository.
